.github/workflows/v3-nightly.yml

```diff
@@ -65,6 +65,27 @@ jobs:
           if [ -d .userbenchmark ]; then rm -Rf .userbenchmark; fi
           python run_benchmark.py torch-nightly -c v3-cuda-tests.yaml
           cp -r ./.userbenchmark/torch-nightly ../benchmark-output
+      - name: Detect potential regressions
+        continue-on-error: true
+        run: |
+          . "${SETUP_SCRIPT}" && conda activate "${CONDA_ENV_NAME}"
+          pushd benchmark
+          RESULTS=($(find ${PWD}/../benchmark-output -name "metrics-*.json" -maxdepth 2 | sort -r))
+          # TODO: the following assumes only one metrics-*.json is found. It will keep
+          # overwriting gh-issue.md if multiple are found.
+          for r in ${RESULTS[@]}; do
+            python regression_detector.py --platform "${PLATFORM_NAME}" --treatment "${r}" --owner @xuzhao9 \
+            --gh-issue-path gh-issue.md --errors-path errors.txt
+          done
+      - name: Create the github issue
+        continue-on-error: true
+        if: env.TORCHBENCH_REGRESSION_DETECTED
+        uses: peter-evans/create-issue-from-file@v4
+        with:
+          title: V2 Performance Signal Detected by TorchBench Userbenchmark "torch-nightly" on ${{ env.TORCHBENCH_REGRESSION_DETECTED }}
```

Comment:
Should this be V2 still? Or V3 haha since it says V3 nightly in the file. Also what is `TORCHBENCH_REGRESSION_DETECTED` supposed to represent?

Reply:
Sorry, it should be V3 instead.
In https://github.com/pytorch/benchmark/blob/main/regression_detector.py#LL134C17-L134C47, we will set TORCHBENCH_REGRESSION_DETECTED to the treatment commit hash.